webui/react/src/stores/workspaces.tsx

```diff
@@ -112,16 +113,21 @@ class WorkspaceStore extends PollingStore {
     );
   }
 
-  public fetch(settings = {} as GetWorkspacesParams, signal?: AbortSignal): () => void {
+  public fetch(signal?: AbortSignal): () => void {
     const canceler = new AbortController();
 
-    getWorkspaces(settings, { signal: signal ?? canceler.signal })
+    getWorkspaces({}, { signal: signal ?? canceler.signal })
       .then((response) => this.#loadableWorkspaces.set(Loaded(response.workspaces)))
       .catch(handleError);
 
     return () => canceler.abort();
   }
 
+  public fetchCached(signal?: AbortSignal): () => void {
```

Comment:
Can you make this consistent with [projects store](https://github.com/determined-ai/determined/blob/864b7e2da349ce63c00a6450a9aa8026c00de723/webui/react/src/stores/projects.tsx#L13)?